src/spring-cloud/azext_spring_cloud/_build_service.py

```diff
@@ -20,3 +21,30 @@ def _update_default_build_agent_pool(cmd, client, resource_group, name, build_po
             properties=build_properties)
         return client.build_service_agent_pool.begin_update_put(
             resource_group, name, DEFAULT_BUILD_SERVICE_NAME, DEFAULT_BUILD_AGENT_POOL_NAME, agent_pool_resource)
+
+
+def create_or_update_builder(cmd, client, resource_group, service, name, builder_json=None, builder_file=None):
+    builder = _update_builder(builder_file, builder_json, [])
+    builder_resource = models_20220101preview.BuilderResource(
+        properties=builder
+    )
+    return client.build_service_builder.begin_create_or_update(resource_group, service, DEFAULT_BUILD_SERVICE_NAME,
+                                                               name, builder_resource)
+
+
+def builder_show(cmd, client, resource_group, service, name,):
+    return client.build_service_builder.get(resource_group, service, DEFAULT_BUILD_SERVICE_NAME, name)
+
+
+def builder_delete(cmd, client, resource_group, service, name):
+    return client.build_service_builder.begin_delete(resource_group, service, DEFAULT_BUILD_SERVICE_NAME, name)
+
+
+def _update_builder(builder_file, builder_json, builder):
+    if builder_file is not None:
+        with open(builder_file, 'r') as json_file:
+            builder = json.load(json_file)
+
+    if builder_json is not None:
+        builder = json.loads(builder_json)
```

Comment:
What if the json is not in the correct format?
When saying the json is "correct", there would be 2 meanings:
1. The Json itself is correct
2. The Json conten can construct a builder's properties

Reply:
1. If the Json itself is incorrect, `json.load` will raise a cli error.
2. If the Json has incorrect properties, these properties will be ignored.